synapse/models/economic.py

```diff
@@ -561,8 +538,7 @@
                 ('business', ('ou:org', {}), {
                     'doc': 'The business which is the registered owner of the SWIFT BIC.'}),
 
-                # FIXME ou:office?
-                ('office', ('entity:actor', {}), {
+                ('office', ('geo:place', {}), {
```

Comment:
Rethink? Should we leave :business but remove :office and let any "branch" specific ID be listed in entity:contact:identifiers? Specifically a "branch" does not have to be an office/physical location: "The branch identifier is a 3 character OPTIONAL element that can supplement the 8 character BIC, used to identify specific locations, departments, services or units of the same business party."

Also, can we revisit the BIC regex? We currently require the 3-char branch ID, but it's optional and not all data (e.g., OFAC sanctions) includes this (means OFAC SWIFT numbers have to be modeled as ou:id / ou:id:number b/c our model is too restrictive.